gtsfm/evaluation/merge_reports.py

```diff
@@ -0,0 +1,107 @@
+"""Merge two GTSFM reports for side-by-side comparison.
+
+Authors: John Lambert
+"""
+
+import argparse
+from collections import defaultdict
+from typing import Any, Dict, List, Tuple
+
+from tabulate import tabulate
+
+GTSFM_REPORT_TABLES = Dict[str, Dict[str, Any]]
+# contains info about each metric, from two separate reports
+GTSFM_MERGED_REPORT_TABLES = Dict[str, List[Tuple[str, Any, Any]]]
```

Comment:
Why does the type of the value differ? I think the only difference should be going from 2-tuple to 3-tuple